src/Table/Table.tsx

```diff
@@ -96,6 +113,8 @@ export function Table<RowShape>({
                       ? `none`
                       : `1px solid ${colors.silver.dark}`,
                   padding,
+                  paddingLeft: colIndex === 0 ? 0 : padding,
```

Comment:
Why are you changing the padding here? I think @mayakoneval suggested you create a storybook story showing how to use the `colProps`, is that what you meant to do?

Reply:
@justinanastos This was a commit @mayakoneval slipped into my PR haha. I assume she didn't want to create a whole new PR. But you're right in that I forgot to add the storybook to demo colProps. Will do that.

Reply:
hi I just wanted to make the the left was left aligned and the right could be right aligned so I'm taking out the padding because in all table designs this is the case!